setup.py

```diff
@@ -297,7 +318,7 @@ def run_setup(extensions):
     kw['ext_modules'] = [Extension('DUMMY', [])]  # dummy extension makes sure build_ext is called for install
 
     if try_cython:
-        kw['setup_requires'] = ['Cython >=0.21']
+        kw['setup_requires'] = ['Cython']
```

Comment:
My only question would be, are there other code compatibility issues with older versions (pre .20) of cython we need to be aware of? Maybe we should still specify a minimum version of Cython?